packages/webdriverio/tests/utils/interception/devtools.test.js

```diff
@@ -156,7 +156,7 @@ test('decodes base64 responses', async () => {
     const mock = new NetworkInterception('**/foobar/**')
     await NetworkInterception.handleRequestInterception(cdpClient, [mock])({
         request: { url: 'http://test.com/foobar/test.html' },
-        responseHeaders: [{ name: 'Content-Type', value: 'application/json' }]
+        responseHeaders: [{ name: 'content-Type', value: 'application/json' }]
```

Comment:
I noticed that you changed the capitalisation of the header here. Given that all headers are not case-sensitive, should this be `content-type` then?